riff-raff.yaml

```diff
@@ -27,7 +29,23 @@ deployments:
         CODE: NewslettersTool-CODE.template.json
         PROD: NewslettersTool-PROD.template.json
 
+        # Newsletters internal tool cloudformation deployment
+  newsletters-api-cfn:
+    template: cloudformation-template
+    app: newsletters-tool
+    parameters:
+      cloudFormationStackName: newsletters-tool
+      amiParameter: AMINewsletterstool # please don't change this unless you know what you're doing
```

Comment:
Interesting comment...

Using https://github.com/guardian/cdk/tree/main/src/experimental/riff-raff-yaml-file should simplify things here.